inc/Engine/Media/Fonts/Frontend/Controller.php

```diff
@@ -104,6 +123,15 @@ protected function get_optimized_markup( string $hash, string $original_url ): s
 
 		$gf_parameters = wp_parse_url( $original_url, PHP_URL_QUERY );
 
+		$internal_styling = wpm_apply_filters_typed( 'boolean', 'rocket_internal_fonts_styling', false );
+		if ( $internal_styling ) {
```

Comment:
```suggestion
		$inline_fonts_css = wpm_apply_filters_typed( 'boolean', 'rocket_host_fonts_locally_inline_css', false );
		if ( $inline_fonts_css ) {
```

Need the docblock too